lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -46,36 +58,34 @@ class Meta:
         ]
 
 
+class ScoreField(serializers.Field):
+    def to_representation(self, value):
+        if ('pointsEarned' not in value) and ('pointsPossible' not in value):
+            return None
+        return ScoreSerializer(value).data
+
+
 class ScoreSerializer(serializers.Serializer):  # pylint: disable=abstract-method
     """
     Score (points earned/possible) for use in SubmissionMetadataSerializer
     """
     pointsEarned = serializers.IntegerField(default=0)
     pointsPossible = serializers.IntegerField(default=0)
 
-    class Meta:
-        fields = ['pointsEarned', 'pointsPossible']
-
-    def to_representation(self, instance):
-        """ An empty dict should return None instead """
-        if ('pointsEarned' not in instance) and ('pointsPossible' not in instance):
-            return None
-        return super().to_representation(instance)
-
 
 class SubmissionMetadataSerializer(serializers.Serializer):  # pylint: disable=abstract-method
     """
     Submission metadata for displaying submissions table in ESG
     """
-    submissionUuid = serializers.CharField()
+    submissionUuid = serializers.UUIDField()
```

Comment:
I remember having to change this to `Char` for some reason, I feel like it broke something in the BFF. Maybe it's _only_ an issue in the BFF but there _was_ a reason I kept it `CharField`

Reply:
Okay, no worries, I'll just put it back